src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -90,9 +98,13 @@ def update_data(self):
         """
         for queue_key, sai_id in self.port_queues_map.items():
             queue_stat_name = mibs.queue_table(sai_id)
-            queue_stat = Namespace.dbs_get_all(self.db_conn, mibs.COUNTERS_DB, queue_stat_name, blocking=False)
+            port_index, _ = queue_key.split(':')
+            queue_stat_idx = mibs.queue_key(port_index, queue_stat_name)
+            namespace = self.port_index_namespace[int(port_index)]
+            queue_stat = self.namespace_db_map[namespace].get_all( \
+                    mibs.COUNTERS_DB, queue_stat_name, blocking=False)
             if queue_stat is not None:
```

Comment:
>if queue_stat is not None [](start = 12, length = 25)

Otherwise, del the index from `self.queue_stat_map`

Reply:
> > if queue_stat is not None [](start = 12, length = 25)
> 
> Otherwise, del the index from `self.queue_stat_map`

Added as per comment.